src/components/AccordionFAQ/AccordionFAQ.style.ts

```diff
@@ -1,85 +1,150 @@
   Accordion as MuiAccordion,
   AccordionDetails as MuiAccordionDetails,
+  AccordionSummary as MuiAccordionSummary,
+  Typography,
   type Breakpoint,
 } from '@mui/material';
 
-import { styled } from '@mui/material/styles';
+import { alpha, keyframes, styled } from '@mui/material/styles';
 
-export const AccordionContainer = styled(Container)(({ theme }) => ({
+export const AccordionBox = styled(Box)(({ theme }) => ({
   margin: 'auto',
-  marginTop: theme.spacing(4),
-  padding: theme.spacing(1, 2),
+  // marginTop: theme.spacing(4),
+  // padding: theme.spacing(1, 2),
   borderRadius: '8px',
-  background: theme.palette.surface1.main,
   position: 'relative',
   maxWidth: theme.breakpoints.values.md,
   width: '100% !important',
-  boxShadow:
-    theme.palette.mode === 'dark'
-      ? '0px 2px 4px rgba(0, 0, 0, 0.04), 0px 8px 16px rgba(0, 0, 0, 0.08)'
-      : '0px 2px 4px rgba(0, 0, 0, 0.04), 0px 8px 16px rgba(0, 0, 0, 0.04)',
+  // boxShadow:
+  //   theme.palette.mode === 'dark'
+  //     ? '0px 2px 4px rgba(0, 0, 0, 0.04), 0px 8px 16px rgba(0, 0, 0, 0.08)'
+  //     : '0px 2px 4px rgba(0, 0, 0, 0.04), 0px 8px 16px rgba(0, 0, 0, 0.04)',
 
-  '&:before': {
-    content: '" "',
-    zIndex: '-1',
-    position: 'absolute',
-    top: 0,
-    right: 0,
-    bottom: 0,
-    left: 0,
-    background: 'linear-gradient(to top left, #fff 0%, #fff 100% )',
-    filter: 'blur(20px)',
-    opacity: 'var(0.7)',
-    transition: 'opacity 0.3s',
-  },
   [theme.breakpoints.up('sm' as Breakpoint)]: {
     width: theme.breakpoints.values.md,
     maxWidth: theme.breakpoints.values.md,
   },
 }));
 
-export interface AccordionProps extends Omit<MuiAccordionProps, 'component'> {
+interface FaqShowMoreArrowProps {
+  arrowSize?: number;
+}
+
+export const rotateAnimation = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(180deg);
+  }
+`;
+
+export const FaqShowMoreArrow = styled(ArrowForwardIosIcon, {
+  shouldForwardProp: (prop) => prop !== 'arrowSize',
+})<FaqShowMoreArrowProps>(({ arrowSize }) => ({
+  width: arrowSize || 24,
+  height: arrowSize || 24,
+  transition: 'transform 0.3s ease',
+  transform: 'rotate(90deg)',
+}));
+
+export interface AccordionItemWrapperProps extends BoxProps {
   show?: boolean;
 }
 
-export const Accordion = styled(MuiAccordion, {
-  shouldForwardProp: (prop) => prop !== 'backgroundImageUrl',
-})<AccordionProps>(({ theme, show }) => ({
-  background: 'transparent',
-  visibility: show ? 'visible' : 'hidden',
-  height: show ? 'auto' : 0,
+export const AccordionItemWrapper = styled(Box)<AccordionItemWrapperProps>(
+  ({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+    gap: theme.spacing(1),
+  }),
+);
+
+export const Accordion = styled(MuiAccordion)(({ theme }) => ({
+  fontFamily: 'inherit',
+  borderRadius: '16px',
+  padding: theme.spacing(2, 3),
+  width: '100%',
+  background: alpha(theme.palette.text.primary, 0.08),
```

Comment:
Would be good to extract it to the theme to avoid duplication

Reply:
This can be a topic for another day, a different ticket to clean this up and manage the general usage of such within the theme.

Reply:
agree